### ISSUE TITLE
Certificate expired at https://software.virtualmin.com

### ISSUE THREAD

**Issue by fernandolcardoso** [virtualmin/virtualmin-install#18]
Hello Virtualmin, I am having trouble to perform the installation, due to an issue of access to your server that has the certificate expired.

![image](https://user-images.githubusercontent.com/16019127/44906595-b93ee900-aceb-11e8-8f42-4c28822fae90.png)

**Comment by d4d4** [comment]
True, the installation doesn't work unless I change the install.sh to use http instead.
